src/redux/slices/educational-instutions/middleware.ts

```diff
@@ -78,10 +83,60 @@ const clearInstitutesList = () => async (dispatch: AppDispatch) => {
   }
 }
 
+const createEducationalInstitutes =
+  (formData: ICreateEducationalInstituteFormDataProps) => async (dispatch: AppDispatch) => {
+    try {
+      dispatch(setCreateIndividualInstitutes(true))
+      await API.educationalInstitutes.createEducationalInstitute(formData)
+    } catch (error) {
+      /* empty */
```

Comment:
set errors here

Reply:
we have global error, in middleware we leave catch empty